src/passes/Print.cpp

```diff
@@ -912,8 +911,15 @@ struct PrintSExpression : public Visitor<PrintSExpression> {
   }
   void visitTable(Table* curr) {
     if (!curr->exists) return;
-    // if table wasn't imported, declare it
-    if (!curr->imported) {
+    if (curr->imported()) {
+      doIndent(o, indent);
+      o << '(';
+      printMedium(o, "import ");
```

Comment:
Yeah, probably a good idea to extract the import header.

Reply:
Done.